content/docs/integrations/adr/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Configuring Architecture Decision Records (ADR)
+publishedDate: '2025-07-25T10:39:00.0Z'
+description: How to configure the ADR plugin to explore Architecture Decision Records in Roadie.
+humanName: ADR
+logoImage: '../../../assets/logos/adr/adr-logo.webp'
+integrationType: OSS plugin
+---
+
+## Introduction
+
+The Architecture Decision Records (ADR) plugin allows you to explore ADRs associated with your entities, as well as discover ADRs across other entities using the Search. Use this to inform your own architectural decisions based on the experiences of previous projects.
+
+This plugin helps teams document and share architectural decisions, making it easier to understand the reasoning behind technical choices and learn from past decisions across different projects.
```

Comment:
It might make sense to introduce the fact that the adr directory structure and adr file structure is important and is standardised and also refer to the stardards upfront.

Something like:

```suggestion
The plugin helps teams share architectural decisions that have been documented using [MADR templates](https://adr.github.io/madr/). Using MADR with Roadie makes it easier to understand the reasoning behind technical choices and learn from past decisions across different projects.
```